imagetest/test_suites/metadata/metadata_test.go

```diff
@@ -1,24 +1,75 @@
+// +build cit
+
 package metadata
 
 import (
 	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
 )
 
+const metadataURLIPPrefix = "http://169.254.169.254/computeMetadata/v1/instance/"
```

Comment:
Can you add a link to the GCP docs regarding this URL?

Reply:
interesting. I don't find a public doc about this. But seems this is a reserver and dedicated address. It's well-known ip.
https://stackoverflow.com/questions/42314029/whats-special-about-169-254-169-254-ip-address-for-aws